justfile

```diff
@@ -32,6 +32,9 @@ check-json: (check '--message-format=json')
 # Run with args
 run *args:
     cargo run {{args}}
+    
+erase-all-configs:
```

Comment:
This actually gave me an idea, installing example/community configs from an `examples` folder.